app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def inner(*args) -> None:
```

Comment:
```suggestion
    def inner(*args) -> Any:
```
inner could return random results of `func`

Reply:
Any could be imported from typing